### DISCUSSION TITLE
[@types/mui-datatables] uses deprecated @material-ui/core & @material-ui/types

### DISCUSSION THREAD

**Discussion by liamlows** [DefinitelyTyped/DefinitelyTyped#58913] [Issues with a @types package]
Hey y'all,

With `@mui` moving to v5, all `@material-*` packages are entering deprecation. Thus `@material-ui/core` & `@material-ui/types` should be updated to use `@mui/material` and `@mui/types`. I will try to perform this change myself and submit a PR but have been quite busy so I figured I would at least create the issue. If anyone would like feel free to jump on this, it shouldn't be too tough to change.

## ISSUE

Use of deprecated packages `@material-ui/core` & `@material-ui/types`.

## REPRODUCTION

N/A, the packages are simply entering deprecation.

## REMEDIATION

Update package.json and imports to use `@mui/material` and `@mui/types`.

Thanks!

[a comment by an automated account is not shown]

**Comment by favna** [comment]
Personally, I'm still reviewing PRs for this package as I feel a responsibility to do so as its creator but I don't really have a personal interest in the library anymore. That said, if you open a PR to make the change that bumps the types to v4 then by all means I will review it, but I don't see myself making the change.

(should also note that like you, I'm also quite busy with all things)

**Reply by liamlows** [reply]
That's fair, I figured that I'd let y'all know first before I go and do it. I'll try to get it done this week since a project at work is using this and it would be nice to remove those deprecated packages from the repo. Thanks for the offer to review, ill convert this to an issue!
